src/client.ts

```diff
@@ -139,9 +140,15 @@ export default class Client extends LanguageClient {
       outputChannel,
     };
 
-    let serverOptions: ServerOptions = {
-      command,
-      args,
+    let serverOptions: ServerOptions = () => {
+      const env = aztecMacroEnabled ? { ...process.env, AZTEC_MACROS: 'true' } : { ...process.env };
```

Comment:
Please help me understand how this will affect LSP? Not obvious if this should affect version from master (I think yes, looking at the issue #52).

Also should this affect LSP only? If no, more lift is needed to support Command, Info and Profile commands.